types/src/keyless/mod.rs

```diff
@@ -80,6 +80,10 @@ macro_rules! serialize {
 pub enum EphemeralCertificate {
     ZeroKnowledgeSig(ZeroKnowledgeSig),
     OpenIdSig(OpenIdSig),
+    ZeroKnowledgeSigV2 {
+        setup_id: String,
+        zk_sig: ZeroKnowledgeSig,
+    },
```

Comment:
@larry-aptos @bowenyang007 here an authenticator change is being proposed. does it break indexer?